src/v2/Apps/Gene/Components/GeneMeta.tsx

```diff
@@ -10,21 +10,21 @@ interface GeneMetaProps {
 
 const GeneMeta: React.FC<GeneMetaProps> = ({ gene }) => {
   const title = `${gene.name} | Artsy`
-  const description = gene.meta.description
+
+  const fallbackDescription =
```

Comment:
**non-blocking**: Might be a nice follow-up refactor sometime to extract this fallback description to a shared location so it can be used across apps.

Reply:
@dblandin thanks for review! I'll bear everything in mind